internal/api/graphql/graph/schema/issue.graphqls

```diff
@@ -13,7 +13,8 @@ type Issue implements Node {
     activities(filter: ActivityFilter, first: Int, after: String): ActivityConnection
     issueMatches(filter: IssueMatchFilter, first: Int, after: String): IssueMatchConnection
     componentVersions(filter: ComponentVersionFilter, first: Int, after: String): ComponentVersionConnection
-    metadata: IssueMetadata
+    issueMetadata: IssueMetadata
+    metadata: Metadata
```

Comment:
would it be possible to have a field metadata that is of type `IssueMetadata` and `IssueMetadata` contains both the issueMetadata and the default metadata?

Reply:
Added refactoring task:
https://github.com/cloudoperators/heureka/issues/369